docs/index.md

```diff
@@ -1,131 +1,250 @@
+| Cross-platform Task Runner | [✅](workspace/advanced_tasks.md "Run shell commands on all platforms with `pixi run`") | ❌ | ❌ | ❌ | ✅ |
+| Lockfiles | [✅](workspace/lockfile.md) | ❌ | ❌ | ✅ | ✅ |
+| Project Management | [✅](reference/pixi_manifest.md) | ❌ | ❌ | ✅ | ✅ |
 
-```toml title="pixi.toml" hl_lines="6-7"
---8<-- "docs/source_files/pixi_workspaces/introduction/task_add/pixi.toml"
-```
+---
 
-After adding the task, you can run the task using its name.
-
-```bash
-pixi run start
-```
 
-```
- __________________
-< Hello Pixi fans! >
- ------------------
-     \   ^__^
-      \  (oo)\_______
-         (__)\       )\/\
-           ||----w |
-           ||     ||
+## 🛠️ Installation
 
-```
+To install `pixi`, run:
 
+=== "Linux & macOS"
+    ```bash
+    curl -fsSL https://pixi.sh/install.sh | sh
+    ```
 
-You now know how to add dependencies and tasks to your environment.
-Put the workspace folder on a different machine, and you will find that Pixi will be able to fully reproduce your setup.
+=== "Windows"
+    ```powershell
+    powershell -ExecutionPolicy ByPass -c "irm -useb https://pixi.sh/install.ps1 | iex"
+    ```
 
-If you want to learn more about Pixi, check out the next page!
+!!! tip "Now restart your terminal or shell!"
+    The installation needs to become effective by restarting your terminal or sourcing your shell.
+
+??? question "Don't trust our link? Check the script!"
+    You can check the installation `sh` script: [download](https://pixi.sh/install.sh) and the `ps1`: [download](https://pixi.sh/install.ps1).
+    The scripts are open source and available on [GitHub](https://github.com/prefix-dev/pixi/tree/main/install).
+
+[**See all installation options →**](installation.md)
+
+---
+
+## 🏁 Getting Started
+
+=== "Python"
+    1. **Initialize a workspace:**
+        ```
+        pixi init hello-world
+        cd hello-world
+        ```
+
+    2. **Add dependencies:**
+        ```
+        pixi add cowpy python
+        ```
+
+    3. **Create your script:**
+        ```py title="hello.py"
+        --8<-- "docs/source_files/pixi_workspaces/introduction/deps_add/hello.py"
+        ```
+
+    5. **Add a task:**
+        ```
+        pixi task add start python hello.py
+        ```
+
+    6. **Run the task:**
+        ```
+        pixi run start
+        ```
+        ```
+        ✨ Pixi task (start): python hello.py
+        __________________
+        < Hello Pixi fans! >
+        ------------------
+            \   ^__^
+            \  (oo)\_______
+                (__)\       )\/\
+                ||----w |
+                ||     ||
+        ```
+
+    7. **Entry the environment shell:**
+        ```
+        pixi shell
+        python hello.py
+        exit
+        ```
+
+    More details on how to use Pixi with Python can be found in the [Python tutorial](python/tutorial.md).
+
+=== "Rust"
```

Comment:
just from a user perspective: for some rust projects, i don't use pixi as i don't get much benefit from it. the rust ecosystem itself is solid enough that you can just install rust using rustup and it easily works.

if i have some non-rust dependencies (for example in pixi-pack i use conda, micromamba for testing), i use pixi in rust projects as well

(for things like cargo-clippy, cargo-insta, ... pixi might be a nice solution as you get the pre-built binaries already into your environment, though)